### DISCUSSION TITLE
Interacting with mods from plugins

### DISCUSSION THREAD

**Discussion by OneByteGH** [IzzelAliz/Arclight#966] [Q&A]
Hey guys, I'm quite new to this concept of mods and plugins running in parallel. I wanted to know if there is a way I can communicate with mods from my plugin. My main target is the [Cobblemon](https://cobblemon.com/) modpack which adds pokemons to the game.

**Comment by Rakise** [comment]
I'm trying too, I can get any values from the mod etc. But I didn't figure out how to listen to forge events in bukkit
